src/facts/field-or-pasture-thistle.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Field or Pasture Thistle
+image:
+  url: https://www.flickr.com/photos/tgpotterfield/14359215078
+  license: CC by-NC-SA 2.0
+  src: ../images/pasturethistle.jpg
+  alt: Field or Pasture Thistle
+  creator: Tom Potterfield
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+category: Plant
```

Comment:
I think we want all category names to be lowercase. Also do we want to have this "plant" category or does it fit "habitat" better?